sunpy/timeseries/sources/fermi_gbm.py

```diff
@@ -59,10 +59,46 @@ class GBMSummaryTimeSeries(GenericTimeSeries):
     # Class attribute used to specify the source class of the TimeSeries.
     _source = 'gbmsummary'
 
+    def plot(self, axes=None, title=None, **kwargs):
```

Comment:
I feel like the API on plot should be the same for all?

Reply:
This changes depending on the number of axes right? This source has only one and isn't deriving it from it's pandas dataframe while EVE/ESP and a few others are

Reply:
Sure but I guess on some level I am uncomfortable for a function that is named the same on a subclass of time series to a different API.

I don't really have a good answer to this.

Reply:
They still return the axes similar to the others right so I thought it wouldn't matter but if that's the case then I'm open to removing the plot functions where the axes are not singular

Reply:
Removing?

Reply:
I meant keeping removing it as it was before this PR

Reply:
Sorry, I've lost track. Removing the axes keywords in plot?

Reply:
Removing this method altogether, if that might be better (for those with multiple axes, but 3 of them will have multiple axes if we decide NOAA to be one as well)

Reply:
You mean removing plot? I think we want to keep plot, in future I plan to push to remove peek.

Reply:
Ah okay, then I shall wait for a better solution to this

Reply:
I think we should remove the title argument, but leave axes.